modules/client/src/tokenVoting/index.ts

```diff
@@ -0,0 +1,9 @@
+export { TokenVotingClient } from "./client";
+export {
+  TokenVotingProposal,
+  TokenVotingProposalListItem,
+  TokenVotingProposalResult,
+  Erc20TokenDetails,
```

Comment:
I guess `Erc20TokenDetails` will be a parallel type to ERC721 and similar in the future, right?

Reply:
yes exactly

Reply:
yes